src/Cli/RestartCommand.php

```diff
@@ -18,8 +21,12 @@ protected function configure()
     {
         $this
             ->setName('docker:restart')
-            ->setDescription('Restart Docker')
-        ;
+            ->setDescription('Restart Docker');
+    }
+
+    public function setProcessRunner(ProcessRunner $processRunner)
+    {
+        $this->processRunner = $processRunner;
```

Comment:
same as above - let me know if you're happy for different constructor signatures